app/gui/src/controller/searcher.rs

```diff
@@ -574,7 +574,8 @@ impl Searcher {
             "Standard.Visualization.AI",
             "build_ai_prompt",
         )?;
-        let vis = Visualization::new(this.id, vis_ptr, vec![]);
+        let vis =
+            Visualization::new("Standard.Visualization.AI".to_string(), this.id, vis_ptr, vec![]);
```

Comment:
This recommendation is from 2015 and is since outdated. Later in the same thread:

> Now that [specialization for str::to_string() 996](https://github.com/rust-lang/rust/pull/32586) has landed, we can safely say that to_string() has the same performance as to_owned()

Now we might have other code style reasons to prefer one over the other, but performance is not one of them.

Reply:
I was just about to do search and replace, but less work is obviously better. Delaying `to_owned` changes for now.